sources/core/objects/mobs/HumanInterface.cpp

```diff
@@ -146,7 +151,7 @@ kv::HumanInterface::HumanInterface()
 
     {
         Button lay;
-        lay.position = {15, 11};
+        lay.position = {15, 12};
```

Comment:
Now the lay-down button and the oxygen indicator share the same place on the screen (15, 12), so the lay down button cannot be seen.
It should be fixed.